packages/react/vite.config.mts

```diff
@@ -79,6 +80,53 @@ export default defineConfig(({ mode }) => {
               }
             },
           },
+          {
+            name: "generate-css-tokens",
```

Comment:
I'm thinking out loud to myself, but I'm wondering if there's an easy way to detect changes to the outputted css vars, via some kind of an automated snapshot test? Since we are constantly iterating and evolving the underlying telegraph system, I think it's possible for those changes to cascade to here and breaking/affecting the out-of-the-box components unintentionally. 

Also not a blocker but maybe worth leaving a TODO to think about.

Reply:
Yeah, agreed this isn't a perfect solution. A few thoughts:
1. The tokens will only change when we upgrade @telegraph/tokens
1. We could manually generate the styles, like move this to a script and just run it and copy the output to `theme.css`. We don't really change the base tokens that often in Telegraph so I think that's not a terrible idea. It'd also be easier to see in the source code what tokens we have.
1. Fully automate generating them. @telegraph/tokens exports an object with all the tokens, so we could use that instead of this somewhat hacky method of manipulating the css.

Reply:
Thanks for thinking through different options! And i'm personally leaning toward option 2, which I imagine isn't that much different in terms of an effort level, but changes will be more transparent? 

> It'd also be easier to see in the source code what tokens we have.

Yeah exactly. 

Again, not a blocker for landing this PR, but it'd be nice to do it before we ship the first version.

Reply:
Yeah, it'll be pretty quick. I'll update the PR to do that instead!